service/src/main/java/bio/terra/workspace/service/workspace/flight/disable/application/DisableApplicationKeys.java

```diff
@@ -0,0 +1,6 @@
+package bio.terra.workspace.service.workspace.flight.disable.application;
+
+public class DisableApplicationKeys {
```

Comment:
style note: most other flight keys are organized as static nested classes inside `WorkspaceFlightMapKeys` instead of standalone files. We should be consistent with that, though I don't have a strong opinion on which approach we take.